en/getting-started/interface.md

```diff
@@ -0,0 +1,46 @@
+---
+id: interface
+title: Discover the Centreon web interface
+---
+
+## First login
+
+To connect to your Centreon web interface, go to URL: `http://IP_ADDRESS/centreon`. (Replace **IP_ADDRESS** by the IP address or FQDN of your Centreon web server.)
+
+Fill in your user name and associated password and click on the **Connect** button:
+
+![image](../assets/getting-started/aconnection.png)
+
+>The default credentials are: `admin`/`centreon`. 
+
+You are now connected to the Centreon web interface.
+
+### Introduction to menus
+
+The Centreon web interface contains several menus, each with a specific function:
```

Comment:
The user can only see this (the picture) if he clocks on the Centreon log in the top left corner (else, he will only see menu icons)